client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -4,6 +4,23 @@ import { useSelector } from 'calypso/state';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import { getSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { ExperimentAssignment } from '@automattic/explat-client';
+import type { ExperimentOptions } from '@automattic/explat-client-react-helpers';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Could use a comment explaining why we're doing this.

Reply:
Done!